examples/fluent-bit/amazon-opensearch-serverless/README.md

```diff
@@ -0,0 +1,5 @@
+### FireLens Example: Logging to Amazon OpenSearch Serverless Service with Fluent Bit
+
+As of aws-for-fluent-bit [version 2.30.0](https://github.com/aws/aws-for-fluent-bit/releases/tag/v2.30.0), Amazon OpenSearch plugin can connect to [Amazon OpenSearch Serverless](https://aws.amazon.com/opensearch-service/). All Amazon OpenSearch plugin Fluent Bit configuration options continue to work as documented when used with Amazon OpenSearch Serverless.
```

Comment:
nit: I think this reads a bit awkward: "All Amazon OpenSearch plugin Fluent Bit configuration options"

Also, I think we can be more clear and helpful for users.

Something like:

----

Amazon OpenSearch Serverless is new offering that eliminates your need to manage OpenSearch instances (or copy their marketing language). All existing Fluent Bit `opensearch` output plugin options work with OpenSearch serverless. The only difference with serverless from a Fluent Bit POV is that you must specify the service name as `aoss` (Amazon OpenSearch Serverless) when you enable `AWS_Auth`:

```
AWS_Auth On
AWS_Region blah
AWS_Service_Name aoss
```